### DISCUSSION TITLE
Azure Portal failed to delete the SAP Monitor

### DISCUSSION THREAD

**Discussion by neil-yechenwei** [Azure/Azure-Monitor-for-SAP-solutions-preview-#54] [General]
I tried to delete the SAP Monitor but it failed with below error. Could you take a look? Thanks.

![image](https://user-images.githubusercontent.com/19754191/232367875-df7cc01c-c1ef-445c-aa0b-df35f2341ef2.png)

**Comment by venu-sivanadham** [comment]
Hi Neil,

The AMS Monitor was created with incomplete properties, especially the ManagedResourceGroupConfig property.
The delete operation looks for this property to do the proper cleanups, since this property is missing the delete operation is failing.

We will rollout a fix and I'll let you know here once its available in production.

Also, please use Microsoft Support Channel to report any issues going forward.
